### DISCUSSION TITLE
Co-Pilot pricing discount typo

### DISCUSSION THREAD

**Discussion by julian-hansen** [community/community#66000] [Copilot Conversations]
### Select Topic Area

General

### Body

At the risk of being pedantic ....

On the pricing page for GitHub CoPilot (https://github.com/github-copilot/signup) the yearly pricing option ($100) claims a 20% discount. This is not correct.

Full price based on monthly would be $10 * 12 = $120. Yearly price is $100. Discount is therefore $ 20/$120 = 16.67% - not 20%

If you were giving a 20% discount then 20% * $120 = $24 resulting in an annual price of $96

Just thought I would point it out.

**Comment by Akash1134** [comment]
Hi @julian-hansen ,

Thank you for sharing the feedback. We have passed it onto the relevant team.
